### ISSUE TITLE
🍀🚨 Horizontal gallery refinement 🚨🍀

### ISSUE THREAD

**Issue by extremobemo** [extremobemo/clover#10]
- Each image should fit inside the screen
- The gallery should be centered on the Y axis and there should be no vertical scroll ever
- Text section needs some work

Mocks attatched
![IMG_0940](https://github.com/user-attachments/assets/f0f03519-cb8a-4828-a2df-f93456bc5378)

**Comment by Tanner-Skelton** [comment]
If you want to tack on the scroll stuff here I can take this one @extremobemo Prolly won't be able to get to it until the weekend though.

**Comment by extremobemo** [comment]
@Tanner-Skelton This ticket is basically done for now. I will need to fire up another ticket with more details once my PR is merged.
